### PR TITLE
Local api requests

### PR DESCRIPTION
**Pull request by Piskvor**
Use loopback for UCRM API requests from plugins to core.

### CODE REVIEW

File: plugins/fio_cz/src/src/Service/UcrmApi.php [hunk 1]
```diff
@@ -32,14 +43,15 @@ public function command(string $endpoint, string $method, array $data): void
...
-        $this->curlExecuter->curlCommand(
+        $this->curlExecutor->curlCommand(
             sprintf('%sapi/v1.0/%s', $optionsData->ucrmPublicUrl, $endpoint),
             $method,
             [
                 'Content-Type: application/json',
                 'X-Auth-App-Key: ' . $optionsData->pluginAppKey,
             ],
-            json_encode((object) $data)
+            json_encode((object)$data),
```
Comment (enumag) [inline plugins/fio_cz/src/src/Service/UcrmApi.php]:
```suggestion
            json_encode((object) $data),
```

File: plugins/quickbooks-online/src/src/Service/UcrmApi.php [hunk 2]
```diff
@@ -41,15 +52,16 @@ public function command(string $endpoint, string $method, array $data): void
                 'Content-Type: application/json',
                 'X-Auth-App-Key: ' . $optionsData->pluginAppKey,
             ],
-            json_encode((object) $data)
+            json_encode((object)$data),
```
Comment (enumag) [inline plugins/quickbooks-online/src/src/Service/UcrmApi.php]:
```suggestion
            json_encode((object) $data),
```

File: plugins/ucrm-client-signup/src/src/initialize.php [hunk 3]
```diff
@@ -8,7 +8,10 @@
 define("UCRM_PUBLIC_URL", $ucrm_json->ucrmPublicUrl);
 define("UCRM_KEY", $ucrm_json->pluginAppKey);
 define("PLUGIN_PUBLIC_URL", $ucrm_json->pluginPublicUrl);
-define("UCRM_API_URL", UCRM_PUBLIC_URL.'api/v2.9');
+$apiUrl = (property_exists($ucrm_json, 'ucrmLocalUrl') && $ucrm_json->ucrmLocalUrl)
+    ? $ucrm_json->ucrmLocalUrl
+    : $ucrm_json->ucrmPublicUrl;
+define("UCRM_API_URL", $apiUrl.'api/v2.9');
```
Comment (enumag) [inline plugins/ucrm-client-signup/src/src/initialize.php]:
```suggestion
define("UCRM_API_URL", $apiUrl.'api/v1.0');
```

**Comment by keksa** [comment]
Also maybe increase versions of all plugins?

**Comment by Piskvor** [comment]
We'll bump with ZIP regen, I guess...

**Pull request merged**